src/renderers/dom/shared/inputValueTracking.js

```diff
@@ -43,15 +47,11 @@ function getTracker(inst: any) {
   return inst._wrapperState.valueTracker;
 }
 
-function attachTracker(inst: InstanceWithWrapperState, tracker: ?ValueTracker) {
-  inst._wrapperState.valueTracker = tracker;
+function detachTracker(subject: SubjectWithWrapperState) {
+  subject._wrapperState.valueTracker = null;
 }
 
-function detachTracker(inst: InstanceWithWrapperState) {
-  delete inst._wrapperState.valueTracker;
-}
-
-function getValueFromNode(node) {
+function getValueFromNode(node: any) {
```

Comment:
In general we should avoid adding `any` wherever possible. It very often leads to bugs. I know we used it sometimes, but I just want to point out for future reviews that it should be used only as last measure.

(I don’t mean this particular case is problematic, but this is something to always keep in mind.)